[MON-21885] [Doc] Bad incrementation in VMware health (#2756)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-vmware2-vcenter-generic.md

| CRITICALTOTALALARMCRITICAL | Critical threshold                                                                                                                                                          |                             |             |
| EXTRAOPTIONS               | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                         | --verbose --ignore-warn-msg |             |

</TabItem>
<TabItem value="Datastore-Io-Global" label="Datastore-Io-Global">

| Macro              | Description                                                                                                                                                | Valeur par défaut | Obligatoire |
|:-------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER             | datastore name to list                                                                                                                                     | .*                |             |

Reviewer: unclear

Author: C'est un filtre sur le nom des databases

| WARNINGWRITE       | Warning threshold                                                                                                                                          |                   |             |
| CRITICALWRITE      | Critical threshold                                                                                                                                         |                   |             |
| EXTRAOPTIONS       | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                        | --verbose         |             |

</TabItem>
<TabItem value="Datastore-Iops-Global" label="Datastore-Iops-Global">

| Macro              | Description                                                                                                                                                | Valeur par défaut | Obligatoire |
|:-------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER             | datastore name to list                                                                                                                                     | .*                |             |

Reviewer: unclear

| WARNINGWRITEVM     | Thresholds                                                                                                                                                 |                   |             |
| CRITICALWRITEVM    | Thresholds                                                                                                                                                 |                   |             |
| EXTRAOPTIONS       | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                        | --verbose         |             |

</TabItem>
<TabItem value="Datastore-Snapshots-Global" label="Datastore-Snapshots-Global">

| Macro            | Description                                                                                                                                                | Valeur par défaut | Obligatoire |
|:-----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER           | datastore name to list                                                                                                                                     | .*                |             |

Reviewer: unclear

| WARNINGTOTAL     | Warning threshold                                                                                                                                          |                   |             |
| CRITICALTOTAL    | Critical threshold                                                                                                                                         |                   |             |
| EXTRAOPTIONS     | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                        | --verbose         |             |

</TabItem>
<TabItem value="Datastore-Usage-Global" label="Datastore-Usage-Global">

| Macro               | Description                                                                                                                                                | Valeur par défaut | Obligatoire |
|:--------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER              | datastore name to list                                                                                                                                     | .*                |             |

Reviewer: unclear

| CRITICALUSAGE       | Thresholds. : 'usage' (B), 'usage-free' (B), 'usage-prct' (%), 'provisioned'                                                                               | 90                |             |
| EXTRAOPTIONS        | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                        | --verbose         |             |

</TabItem>
<TabItem value="Datastore-Vm-Count-Global" label="Datastore-Vm-Count-Global">

| Macro                  | Description                                                                                                                                                | Valeur par défaut | Obligatoire |
|:-----------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER                 | datastore name to check                                                                                                                                    | .*                |             |

Reviewer: unclear

| CRITICALTOTALSUSPENDED | Critical threshold                                                                                                                                         |                   |             |
| EXTRAOPTIONS           | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                        | --verbose         |             |

</TabItem>
<TabItem value="ESX-Alarms-Global" label="ESX-Alarms-Global">

| Macro                      | Description                                                                                                                                                                 | Valeur par défaut           | Obligatoire |
|:---------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------|:-----------:|

> | FILTER                     | ESX hostname to check. If not set, we check all ESX                                                                                                                         | .*                          |             |

Reviewer: unclear

Author: C'est un filtre sur les noms ESX hostname

> | FILTERTIME                 | Don't check alarm older (value in seconds)                                                                                                                                  | 3600                        |             |

Reviewer: older than what?

| CRITICALTOTALALARMCRITICAL | Critical threshold                                                                                                                                                          |                             |             |
| EXTRAOPTIONS               | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                         | --verbose --ignore-warn-msg |             |

</TabItem>
<TabItem value="Esx-Cpu-Global" label="Esx-Cpu-Global">

| Macro               | Description                                                                                                                                          | Valeur par défaut | Obligatoire |
|:--------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER              | ESX hostname to check. If not set, we check all ESX                                                                                                  | .*                |             |

Reviewer: unclear

| WARNINGTOTALCPUMHZ  | Warning threshold                                                                                                                                    |                   |             |
| CRITICALTOTALCPUMHZ | Critical threshold                                                                                                                                   |                   |             |
| EXTRAOPTIONS        | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                  | --verbose         |             |

</TabItem>
<TabItem value="Esx-Datastores-Latency-Global" label="Esx-Datastores-Latency-Global">

| Macro                | Description                                                                                                                                          | Valeur par défaut | Obligatoire |
|:---------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTERESXNAME        | ESX hostname to check. If not set, we check all ESX                                                                                                  | .*                |             |

Reviewer: unclear

> | FILTERDATASTORENAME  | Datastore to check. If not set, we check all datastores                                                                                              | .*                |             |

Reviewer: unclear

| WARNINGWRITELATENCY  | Warning threshold                                                                                                                                    |                   |             |
| CRITICALWRITELATENCY | Critical threshold                                                                                                                                   |                   |             |
| EXTRAOPTIONS         | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                  | --verbose         |             |

</TabItem>
<TabItem value="Esx-Health-Global" label="Esx-Health-Global">

| Macro                  | Description                                                                                                                                          | Valeur par défaut           | Obligatoire |
|:-----------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------|:-----------:|

> | FILTER                 | ESX hostname to check. If not set, we check all ESX                                                                                                  | .*                          |             |

Reviewer: unclear

| CRITICALTOTALPROBLEMS  | Thresholds                                                                                                                                           |                             |             |
| EXTRAOPTIONS           | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                  | --verbose                   |             |

</TabItem>
<TabItem value="Esx-Memory-Global" label="Esx-Memory-Global">

| Macro                  | Description                                                                                                                                          | Valeur par défaut | Obligatoire |
|:-----------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER                 | ESX hostname to check. If not set, we check all ESX                                                                                                  | .*                |             |

Reviewer: unclear

| UNKNOWNSTATUS          | Define the conditions to match for the status to be UNKNOWN (Default: '%{status} !~ /^connected$/i'). You can use the following variables: %{status} |                   |             |
| WARNING                | Warning threshold (can use unit option)                                                                                                              |                   |             |
| CRITICAL               | Critical threshold (can use unit option)                                                                                                             |                   |             |

> | WARNINGOVERHEADMEMORY  | Threshold overhead                                                                                                                                   |                   |             |

Reviewer: ```suggestion
| WARNINGOVERHEADMEMORY  | Overhead threshold                                                                                                                                  |                   |             |
```

| CRITICALSTATUS         | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{status}                           |                   |             |
| EXTRAOPTIONS           | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                  | --verbose         |             |

</TabItem>
<TabItem value="Esx-Service-Global" label="Esx-Service-Global">

| Macro                 | Description                                                                                                                                                                                   | Valeur par défaut                             | Obligatoire |
|:----------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------|:-----------:|

> | FILTER                | ESX hostname to check. If not set, we check all ESX                                                                                                                                           | .*                                            |             |

Reviewer: unclear

| CRITICALSTATUS        | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{status}                                                                    |                                               |             |
| EXTRAOPTIONS          | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                                           | --verbose                                     |             |

</TabItem>
<TabItem value="Esx-Status-Global" label="Esx-Status-Global">

| Macro                 | Description                                                                                                                                                       | Valeur par défaut               | Obligatoire |
|:----------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------|:-----------:|

> | FILTER                | ESX hostname to check. If not set, we check all ESX                                                                                                               | .*                              |             |

Reviewer: unclear

| CRITICALSTATUS        | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{status}                                        |                                 |             |
| EXTRAOPTIONS          | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                               | --verbose                       |             |

</TabItem>
<TabItem value="Esx-Storage-Global" label="Esx-Storage-Global">

| Macro                   | Description                                                                                                                                                                                        | Valeur par défaut                       | Obligatoire |
|:------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------|:-----------:|

> | FILTER                  | ESX hostname to check. If not set, we check all ESX                                                                                                                                                | .*                                      |             |

Reviewer: unclear

| WARNINGADAPTERSTOTAL    | Thresholds                                                                                                                                                                                         |                                         |             |
| CRITICALADAPTERSTOTAL   | Thresholds                                                                                                                                                                                         |                                         |             |
| WARNINGADAPTERSUNKNOWN  | Thresholds                                                                                                                                                                                         |                                         |             |
| CRITICALADAPTERSUNKNOWN | Thresholds                                                                                                                                                                                         |                                         |             |

> | WARNINGLUNSDEGRADED     |                                                                                                                                                                                                    |                                         |             |

Reviewer: no description

| CRITICALSTATUS          | Define the conditions to match for the status to be CRITICAL. You can use the following variables: %{status}, %{maintenance}                                                                       |                                         |             |
| EXTRAOPTIONS            | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                                                | --verbose                               |             |

</TabItem>
<TabItem value="Esx-Swap-Global" label="Esx-Swap-Global">

| Macro           | Description                                                                                                                                          | Valeur par défaut | Obligatoire |
|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER          | ESX hostname to check. If not set, we check all ESX                                                                                                  | .*                |             |

Reviewer: unclear

| WARNINGSWAPOUT  | Warning threshold                                                                                                                                    |                   |             |
| CRITICALSWAPOUT | Critical threshold                                                                                                                                   |                   |             |
| EXTRAOPTIONS    | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                  | --verbose         |             |

</TabItem>
<TabItem value="Esx-Time-Global" label="Esx-Time-Global">

| Macro          | Description                                                                                                                                          | Valeur par défaut | Obligatoire |
|:---------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER         | ESX hostname to check. If not set, we check all ESX                                                                                                  | .*                |             |

Reviewer: unclear

| WARNINGTIME    | Warning threshold in seconds                                                                                                                         | -2:2              |             |
| CRITICALTIME   | Critical threshold in seconds                                                                                                                        | -5:5              |             |
| EXTRAOPTIONS   | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                  | --verbose         |             |

</TabItem>
<TabItem value="Esx-Traffic-Global" label="Esx-Traffic-Global">

| Macro                  | Description                                                                                                                                                         | Valeur par défaut | Obligatoire |
|:-----------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTERESXNAME          | ESX hostname to check. If not set, we check all ESX                                                                                                                 | .*                |             |

Reviewer: unclear

| CRITICALSTATUS         | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{status}                                          |                   |             |
| EXTRAOPTIONS           | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                 | --verbose         |             |

</TabItem>
<TabItem value="Esx-Uptime-Global" label="Esx-Uptime-Global">

| Macro          | Description                                                                                                                                          | Valeur par défaut | Obligatoire |
|:---------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER         | ESX hostname to check. If not set, we check all ESX                                                                                                  | .*                |             |

Reviewer: unclear

| WARNINGTIME    | Warning threshold in seconds                                                                                                                         |                   |             |
| CRITICALTIME   | Critical threshold in seconds                                                                                                                        |                   |             |
| EXTRAOPTIONS   | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                  | --verbose         |             |

</TabItem>
<TabItem value="Esx-Vm-Count-Global" label="Esx-Vm-Count-Global">

| Macro                  | Description                                                                                                                                          | Valeur par défaut | Obligatoire |
|:-----------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER                 | ESX hostname to check. If not set, we check all ESX                                                                                                  | .*                |             |

Reviewer: unclear

| CRITICALTOTALSUSPENDED | Critical threshold                                                                                                                                   |                   |             |
| EXTRAOPTIONS           | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                  | --verbose         |             |

</TabItem>
<TabItem value="Esx-is-Maintenance-Global" label="Esx-is-Maintenance-Global">

| Macro                     | Description                                                                                                                                              | Valeur par défaut         | Obligatoire |
|:--------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------|:-----------:|

> | FILTER                    | ESX hostname to check. If not set, we check all ESX                                                                                                      | .*                        |             |

Reviewer: unclear

| WARNINGSTATUS             | Define the conditions to match for the status to be WARNING (Default: ''). You can use the following variables: %{status}                                |                           |             |
| CRITICALSTATUS            | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{status}                               |                           |             |
| EXTRAOPTIONS              | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                      | --verbose                 |             |

</TabItem>
<TabItem value="Licenses" label="Licenses">

| Macro                 | Description                                                                                                                                | Valeur par défaut | Obligatoire |
|:----------------------|:-------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | UNIT                  | Select the unit for expires threshold. May be 's' for seconds,'m' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is days | d                 | X           |

Reviewer: expires?

Author: C'est un filtre courant pour le mode uptime qui est commun à plusieurs connecteurs vraisemblablement (snmp standard > uptime).
Dans le mode commun l'option est référencée comme ceci :
```
=item B<--unit>

Select the unit for performance data and thresholds. May be 's' for seconds, 'm' for minutes,
'h' for hours, 'd' for days, 'w' for weeks.  Default is seconds
```
https://github.com/centreon/centreon-plugins/blob/c9831fb55c905740322e949d63e612ca5c2d9371/src/snmp_standard/mode/uptime.pm

| CRITICALUSAGEPRCT     | Thresholds                                                                                                                                 |                   |             |
| EXTRAOPTIONS          | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                        | --verbose         |             |

</TabItem>
<TabItem value="Vm-Cpu-Global" label="Vm-Cpu-Global">

| Macro            | Description                                                                                                                                                                                                                      | Valeur par défaut | Obligatoire |
|:-----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER           | VM hostname to check. If not set, we check all VMs                                                                                                                                                                               | .*                |             |

Reviewer: unclear

| EXTRAOPTIONS     | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                                                                              | --verbose         |             |

</TabItem>
<TabItem value="Vm-Datastores-Iops-Global" label="Vm-Datastores-Iops-Global">

| Macro                   | Description                                                                                                                                                                                                                      | Valeur par défaut | Obligatoire |
|:------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER                  | VM hostname to check. If not set, we check all VMs                                                                                                                                                                               | .*                |             |

Reviewer: unclear

| CRITICALWRITE           | Thresholds                                                                                                                                                                                                                       |                   |             |
| EXTRAOPTIONS            | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                                                                              | --verbose         |             |

</TabItem>
<TabItem value="Vm-Device-Global" label="Vm-Device-Global">

| Macro                        | Description                                                                                                                                                                | Valeur par défaut | Obligatoire |
|:-----------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER                       | VM hostname to check. If not set, we check all VMs                                                                                                                         | .*                |             |

Reviewer: unclear

| CRITICALTOTALDEVICECONNECTED | Critical threshold                                                                                                                                                         |                   |             |
| EXTRAOPTIONS                 | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                        | --verbose         |             |

</TabItem>
<TabItem value="Vm-Limit-Global" label="Vm-Limit-Global">

| Macro                | Description                                                                                                                                                                                                                | Valeur par défaut | Obligatoire |
|:---------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER               | VM hostname to check. If not set, we check all VMs                                                                                                                                                                         | .*                |             |

Reviewer: unclear

| EXTRAOPTIONS         | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                                                                        | --verbose         |             |

</TabItem>
<TabItem value="Vm-Memory-Global" label="Vm-Memory-Global">

| Macro              | Description                                                                                                                                                                                                                      | Valeur par défaut | Obligatoire |
|:-------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER             | VM hostname to check. If not set, we check all VMs                                                                                                                                                                               | .*                |             |

Reviewer: unclear

| CRITICALSTATUS     | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{connection\_state}, %{power\_state}                                                                           |                   |             |
| EXTRAOPTIONS       | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                                                                              | --verbose         |             |

</TabItem>
<TabItem value="Vm-Snapshot-Global" label="Vm-Snapshot-Global">

| Macro        | Description                                                                                         | Valeur par défaut                                     | Obligatoire |
|:-------------|:----------------------------------------------------------------------------------------------------|:------------------------------------------------------|:-----------:|

> | FILTER       | VM hostname to check. If not set, we check all VMs                                                  | .*                                                    |             |

Reviewer: unclear

| CRITICAL     | Critical threshold for snapshot's age                                                               | 432000                                                |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) | --disconnect-status='ok' --nopoweredon-skip --verbose |             |

</TabItem>
<TabItem value="Vm-Status-Global" label="Vm-Status-Global">

| Macro                 | Description                                                                                                                                                                                 | Valeur par défaut               | Obligatoire |
|:----------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------|:-----------:|

> | FILTER                | VM hostname to check. If not set, we check all VMs                                                                                                                                          | .*                              |             |

Reviewer: unclear

| CRITICALSTATUS        | Define the conditions to match for the status to be CRITICAL. You can use the following variables: %{connection\_state}, %{power\_state}                                                    |                                 |             |
| EXTRAOPTIONS          | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                                         | --verbose                       |             |

</TabItem>
<TabItem value="Vm-Swap-Global" label="Vm-Swap-Global">

| Macro           | Description                                                                                                                                                                                                                      | Valeur par défaut | Obligatoire |
|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER          | VM hostname to check. If not set, we check all VMs                                                                                                                                                                               | .*                |             |

Reviewer: unclear

| CRITICALSWAPOUT | Critical threshold                                                                                                                                                                                                               |                   |             |
| EXTRAOPTIONS    | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                                                                              | --verbose         |             |

</TabItem>
<TabItem value="Vm-Thinprovisioning-Global" label="Vm-Thinprovisioning-Global">

| Macro        | Description                                                                                         | Valeur par défaut                                     | Obligatoire |
|:-------------|:----------------------------------------------------------------------------------------------------|:------------------------------------------------------|:-----------:|

> | FILTER       | VM hostname to check. If not set, we check all VMs                                                  | .*                                                    |             |

Reviewer: unclear

| STATUS       | Thinprovisioning status (default: none) Example: 'active,CRITICAL' or 'notactive,WARNING'           | active,WARNING                                        | X           |
| VMUUID       |                                                                                                     |                                                       |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) | --disconnect-status='ok' --nopoweredon-skip --verbose |             |

</TabItem>
<TabItem value="Vm-Tools-Global" label="Vm-Tools-Global">

| Macro              | Description                                                                                         | Valeur par défaut                                     | Obligatoire |
|:-------------------|:----------------------------------------------------------------------------------------------------|:------------------------------------------------------|:-----------:|

> | FILTER             | VM hostname to check. If not set, we check all VMs                                                  | .*                                                    |             |

Reviewer: uncleear

Author: ```
=head1 MODE

Check virtual machine tools.

=item B<--vm-hostname>

VM hostname to check.
If not set, we check all VMs
```
C'est un filtre sur le nom des VM

| NOTINSTALLEDSTATUS | Status if vmtools is not installed (default: critical)                                              | critical                                              | X           |
| NOTRUNNINGSTATUS   | Status if vmtools is not running (default: critical)                                                | critical                                              | X           |

> | NOTUP2DATESTATUS   | Status if vmtools is not upd2date (default: warning)                                                | warning                                               |             |

Reviewer: ```suggestion
| NOTUP2DATESTATUS   | Status if vmtools is not up to date (default: warning)                                                | warning                                               |             |
```

| EXTRAOPTIONS                   | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |

</TabItem>
</Tabs>

3. [Déployez la configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). Le service apparaît dans la liste des services supervisés, et dans la page **Statut des ressources**. La commande envoyée par le connecteur est indiquée dans le panneau de détails du service : celle-ci montre les valeurs des macros.

## Comment puis-je tester le plugin et que signifient les options des commandes ?

Une fois le plugin installé, vous pouvez tester celui-ci directement en ligne
de commande depuis votre collecteur Centreon en vous connectant avec
l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
que le connecteur arrive bien à superviser une ressource en utilisant une commande
telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :

```bash
/usr/lib/centreon/plugins/centreon_vmware_connector_client.pl \

> 	--plugin=apps::vmware::connector::plugin \

Reviewer: the old command with its explanation was interesting

| CRITICALTOTALALARMCRITICAL | Critical threshold                                                                                                                                                          |                             |             |
| EXTRAOPTIONS               | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                         | --verbose --ignore-warn-msg |             |

</TabItem>
<TabItem value="Datastore-Io-Global" label="Datastore-Io-Global">

| Macro              | Description                                                                                                                                                | Valeur par défaut | Obligatoire |
|:-------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER             | datastore name to list                                                                                                                                     | .*                |             |

Reviewer: ```suggestion
| FILTER             | The connector will only take into account alerts coming from the datastores listed here                                                                                                                                      | .*                |             |
```

| WARNINGWRITE       | Warning threshold                                                                                                                                          |                   |             |
| CRITICALWRITE      | Critical threshold                                                                                                                                         |                   |             |
| EXTRAOPTIONS       | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                        | --verbose         |             |

</TabItem>
<TabItem value="Datastore-Iops-Global" label="Datastore-Iops-Global">

| Macro              | Description                                                                                                                                                | Valeur par défaut | Obligatoire |
|:-------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER             | datastore name to list                                                                                                                                     | .*                |             |

Reviewer: ```suggestion
| FILTER             | The connector will only take into account alerts coming from the datastores listed here                                                                                                                                     | .*                |             |
```

| WARNINGWRITEVM     | Thresholds                                                                                                                                                 |                   |             |
| CRITICALWRITEVM    | Thresholds                                                                                                                                                 |                   |             |
| EXTRAOPTIONS       | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                        | --verbose         |             |

</TabItem>
<TabItem value="Datastore-Snapshots-Global" label="Datastore-Snapshots-Global">

| Macro            | Description                                                                                                                                                | Valeur par défaut | Obligatoire |
|:-----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER           | datastore name to list                                                                                                                                     | .*                |             |

Reviewer: ```suggestion
| FILTER           | The connector will only take into account alerts coming from the datastores listed here                                                                                                                                   | .*                |             |
```

| WARNINGTOTAL     | Warning threshold                                                                                                                                          |                   |             |
| CRITICALTOTAL    | Critical threshold                                                                                                                                         |                   |             |
| EXTRAOPTIONS     | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                        | --verbose         |             |

</TabItem>
<TabItem value="Datastore-Usage-Global" label="Datastore-Usage-Global">

| Macro               | Description                                                                                                                                                | Valeur par défaut | Obligatoire |
|:--------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER              | datastore name to list                                                                                                                                     | .*                |             |

Reviewer: ```suggestion
| FILTER              | The connector will only take into account alerts coming from the datastores listed here                                                                                                                                     | .*                |             |
```

| CRITICALUSAGE       | Thresholds. : 'usage' (B), 'usage-free' (B), 'usage-prct' (%), 'provisioned'                                                                               | 90                |             |
| EXTRAOPTIONS        | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                        | --verbose         |             |

</TabItem>
<TabItem value="Datastore-Vm-Count-Global" label="Datastore-Vm-Count-Global">

| Macro                  | Description                                                                                                                                                | Valeur par défaut | Obligatoire |
|:-----------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER                 | datastore name to check                                                                                                                                    | .*                |             |

Reviewer: ```suggestion
| FILTER                 | The connector will only take into account alerts coming from the datastores listed here                                                                                                                                   | .*                |             |
```

| EXTRAOPTIONS           | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                        | --verbose         |             |

</TabItem>
<TabItem value="ESX-Alarms-Global" label="ESX-Alarms-Global">

| Macro                      | Description                                                                                                                                                                 | Valeur par défaut           | Obligatoire |
|:---------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------|:-----------:|
| FILTER                     | ESX hostname to check. If not set, we check all ESX                                                                                                                         | .*                          |             |

> | FILTERTIME                 | Don't check alarm older (value in seconds)                                                                                                                                  | 3600                        |             |

Reviewer: ```suggestion
| FILTERTIME                 | The connector will ignore any alert older than the time period specified here (in seconds).                                                                                                                                 | 3600                        |             |
```

| --warning-*            | Warning threshold. Can be: 'total-alarm-warning', 'total-alarm-critical'.                                                                                                                                                                     |
| --critical-*           | Critical threshold. Can be: 'total-alarm-warning', 'total-alarm-critical'.                                                                                                                                                                    |

</TabItem>
<TabItem value="Datastore-Io-Global" label="Datastore-Io-Global">

| Option             | Description                                                                                                                                                  |
|:-------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | --datastore-name   | datastore name to list.                                                                                                                                      |

Reviewer: ```suggestion
| --datastore-name   | The connector will only take into account alerts coming from the datastores listed here                                                                                                                                       |
```

| --warning-status   | Define the conditions to match for the status to be WARNING (Default: ''). You can use the following variables: %{accessible}                                |
| --critical-status  | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{accessible}                               |
| --warning-*        | Warning threshold. Can be: 'total-read', 'total-write', 'read', 'write'.                                                                                     |
| --critical-*       | Critical threshold. Can be: 'total-read', 'total-write', 'read', 'write'.                                                                                    |

</TabItem>
<TabItem value="Datastore-Iops-Global" label="Datastore-Iops-Global">

| Option                   | Description                                                                                                                                                  |
|:-------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | --datastore-name         | datastore name to list.                                                                                                                                      |

Reviewer: ```suggestion
| --datastore-name         | The connector will only take into account alerts coming from the datastores listed here                                                                                                                                      |
```

| --warning-status         | Define the conditions to match for the status to be WARNING (Default: ''). You can use the following variables: %{accessible}                                |
| --critical-status        | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{accessible}                               |
| --warning-* --critical-* | Thresholds. Can be: 'read-total', 'write-total', 'read', 'write', 'read-vm', 'write-vm'.                                                                     |

</TabItem>
<TabItem value="Datastore-Snapshots-Global" label="Datastore-Snapshots-Global">

| Option             | Description                                                                                                                                                  |
|:-------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | --datastore-name   | datastore name to list.                                                                                                                                      |

Reviewer: ```suggestion
| --datastore-name   | The connector will only take into account alerts coming from the datastores listed here                                                                                                                                     |
```

| --warning-status   | Define the conditions to match for the status to be WARNING (Default: ''). You can use the following variables: %{accessible}                                |
| --critical-status  | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{accessible}                               |
| --warning-*        | Warning threshold. Can be: 'total', 'snapshot'.                                                                                                              |
| --critical-*       | Critical threshold. Can be: 'total', 'snapshot'.                                                                                                             |

</TabItem>
<TabItem value="Datastore-Usage-Global" label="Datastore-Usage-Global">

| Option                   | Description                                                                                                                                                  |
|:-------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | --datastore-name         | datastore name to list.                                                                                                                                      |

Reviewer: ```suggestion
| --datastore-name         | The connector will only take into account alerts coming from the datastores listed here                                                                                                                                    |
```

| --warning-status         | Define the conditions to match for the status to be WARNING (Default: ''). You can use the following variables: %{accessible}                                |
| --critical-status        | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{accessible}                               |
| --warning-* --critical-* | Thresholds. Can be: Can be: 'usage' (B), 'usage-free' (B), 'usage-prct' (%), 'provisioned'.                                                                  |

</TabItem>
<TabItem value="Datastore-Vm-Count-Global" label="Datastore-Vm-Count-Global">

| Option             | Description                                                                                                                                                  |
|:-------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | --datastore-name   | datastore name to check.                                                                                                                                     |

Reviewer: ```suggestion
| --datastore-name   | The connector will only take into account alerts coming from the datastores listed here                                                                                                                                     |
```

| --esx-hostname         | ESX hostname to check. If not set, we check all ESX.                                                                                                                                                                                          |
| --filter               | Datacenter is a regexp.                                                                                                                                                                                                                       |
| --scope-datacenter     | Search in following datacenter(s) (can be a regexp).                                                                                                                                                                                          |
| --scope-cluster        | Search in following cluster(s) (can be a regexp).                                                                                                                                                                                             |

> | --filter-time          | Don't check alarm older (value in seconds).                                                                                                                                                                                                   |

Reviewer: ```suggestion
| --filter-time          | The connector will ignore any alert older than the time period specified here (in seconds).                                                                                                                                                                                                  |
```
